addons/wowl/static/src/components/crash_manager/crash_manager.ts

```diff
@@ -0,0 +1,111 @@
+import { Component, tags, hooks } from "@odoo/owl";
+import { DialogParams, OdooEnv } from "../../types";
+import { Dialog } from "../dialog/dialog";
+import { RPCError, RPCServerError } from "../../services/rpc";
+import { Stringifiable, _lt } from "../../core/localization";
+
+const { useState } = hooks;
+
+const odooExceptionTitleMap: { [name: string]: Stringifiable } = {
```

Comment:
what about defining this inside the crash manager? That way you could use env._t